barman/command_wrappers.py

```diff
@@ -880,6 +884,61 @@ class PgBaseBackup(PostgreSQLClient):
 
     COMMAND_ALTERNATIVES = ["pg_basebackup"]
 
+    def _get_compression_args(
```

Comment:
This does belong in the command wrapper because it's converting the compression options into CLI args specific to this command. The code itself is a bit hard to follow with all the conditionals but in principle I think it's ok here.